### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by adudenamedruby** [dreamsofcode-io/home-row-mods#3]
I've created my config file; and the plist file, and placed in `/Library/LaunchAgents` (as I only want it to run when I log in)
I've then done a `launchctl enable gui/501/com.example.kanata` and `launchctl kickstart gui/501/com.example.kanata` But I can't seem to get kanata to run on my laptop (macos 14.7). I'm currently using Karabiner, but I'd really like to do home-row mods, which karabiner is pretty rough for.

the problem with the instructions is that `launchctl load` has become legacy.

Any advice?

**Comment by JirR02** [comment]
Hey @adudenamedruby,

I was able to follow the steps @elliottminns described in the README file. One thing he did not mention is the macOS's  strict TCC (Transparency, Consent and Control), which is mentioned here in an issue in the kanata repo. (jtroo/kanata/issues/1211) Maybe have have a look at this.

**Comment by elliottminns** [comment]
Hey @adudenamedruby ,

Have you been able to get Kanata working by running it manually in the terminal? 

Most likely @JirR02 is correct as MacOS can be a little strict.

**Comment by adudenamedruby** [comment]
Heya. I eventually locked my computer from being able to use the keyboard when logged in at all, so I had to painfully fix that using the onscreen keyboard lolo so I stopped trying to get it to work. I'll return to trying to get it to work during the holiday break, when I actually have time to try and figure it out properly, haha. For now, I'mma just deal with no home row mods.

Re TCC, I had all that set up properly. I think what I might need to do next is just uninstall karabiner, and do the whole process from scratch.

Tyvm for the replies, though! All the best!
